### ISSUE TITLE
when convert keras model to tflite .. gives me this !!

### ISSUE THREAD

**Issue by dyna22** [tensorflow/tensorflow#51994]
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: ADD, CONV_2D, DEPTHWISE_CONV_2D, FULLY_CONNECTED, MAX_POOL_2D, MUL, SOFTMAX. Here is a list of operators for which you will need custom implementations: Elu.

**Comment by abattery** [comment]
https://www.tensorflow.org/lite/guide/ops_select

Please convert with the Select TF option as suggested.
